modules/vectra.py

```diff
@@ -896,17 +1632,643 @@ def advanced_search(self, stype=None, page_size=50, query=None):
+    def get_internal_networks(self):
+        """
+        Get all internal networks configured on the brain
+        """
+        return requests.get('{url}/settings/internal_network'.format(url=self.url),
+            headers=self.headers, verify=self.verify)
+
+    @validate_api_v2
+    @request_error_handler
+    def set_internal_networks(self, include=[], exclude=[], drop=[], append=True):
+        """
+        Get all internal networks configured on the brain
+        Set account tags
+        :param include: list of subnets to add  the internal subnets list
+        :param exclude: list of subnets to exclude from the internal subnets list
+        :param drop: list of subnets to add to the drop list
+        :param append: overwrites existing lists if set to False, appends to existing tags if set to True
+        """
+        if append and all(isinstance(i, list) for i in [include, exclude, drop]):
+            current_list = self.get_internal_networks().json()
+            # We must make all entries unique
+            payload = {
+                'include': list(set(include).union(set(current_list['included_subnets']))),
+                'exclude': list(set(exclude).union(set(current_list['excluded_subnets']))),
+                'drop': list(set(drop).union(set(current_list['dropped_subnets'])))
+            }
+        elif all(isinstance(i, list) for i in [include, exclude, drop]):
+            payload = {
+                'include': include,
+                'exclude': exclude,
+                'drop': drop
+            }
+        else:
+            raise TypeError('subnets must be of type list')
+
+        return requests.post('{url}/settings/internal_network'.format(url=self.url),
+            json=payload, headers=self.headers, verify=self.verify)
+
+    # TODO see if check parameter has been fixed - APP-10753
+    @request_error_handler
+    def get_health_check(self, check=None):
+        """
+        Get health statistics for the appliance
+        :param check: specific check to run - optional
+        CAUTION: the check parameter is broken for the time being
+        """
+        if not check:
+            return requests.get('{url}/health'.format(url=self.url), headers=self.headers, verify=self.verify)
+        else:
+            if not isinstance(check, str):
+                raise ValueError('check need to be a string')
+            return requests.get('{url}/health/{check}'.format(url=self.url, check=check), headers=self.headers, verify=self.verify)
+        
+
+class VectraClientV2_1(VectraClient):
+
+    def __init__(self, url=None, token=None, verify=False):
+        """
+        Initialize Vectra client
+        :param url: IP or hostname of Vectra brain (ex https://www.example.com) - required
+        :param token: API token for authentication when using API v2*
+        :param verify: Verify SSL (default: False) - optional
+        """
+        super().__init__(url=url, token=token, verify=verify)
```

Comment:
We should be consistent with our super calls, this is the python 3 support way of calling super.  This does not work on python 2.  There are other cases in this file that we are using the python 2 super, which works both on python 2/3.

`super(VectraClientV2_1, self).__init__(url=url, token=token, verify=verify)`